codebase-pipelines/buildspec.yml

```diff
@@ -0,0 +1,48 @@
+version: 0.2
+
+env:
+  parameter-store:
+    SLACK_CHANNEL_ID: /codebuild/slack_oauth_channel
+    SLACK_TOKEN: /codebuild/slack_oauth_token
```

Comment:
How easy would it be to make [[DBTP-1208 Allow configuring SLACK_CHANNEL_ID for each codebase pipeline](https://uktrade.atlassian.net/browse/DBTP-1208) go away while you're reworking these?

Reply:
Shouldn't be too difficult, I'll pull it into squad 1's todo